src/control/cmd/daos/pretty/pool_test.go

```diff
@@ -371,12 +371,42 @@ Pool space info:
 	}
 }
 
-// TODO DAOS-18128: Add more test cases
-//func TestPretty_PrintPoolSelfHealDisable(t *testing.T) {
-//	for name, tc := range map[string]struct {
-//		sysSelfHeal string
-//		poolSelfHeal string
-//}
+func TestPretty_PrintPoolSelfHealDisable(t *testing.T) {
+	for name, tc := range map[string]struct {
+		poolSelfHeal string
+		sysSelfHeal  string
+		expPrintStr  string
+	}{
+		"defaults": {
+			poolSelfHeal: "exclude;rebuild",
+			sysSelfHeal:  "exclude;pool_exclude;pool_rebuild",
+		},
+		"no pool flags": {
+			poolSelfHeal: "none",
+			sysSelfHeal:  "exclude;pool_exclude;pool_rebuild",
+			expPrintStr:  "exclude disabled on pool due to [pool] policy\nrebuild disabled on pool due to [pool] policy\n",
+		},
+		"no system flags": {
+			poolSelfHeal: "exclude;rebuild",
+			sysSelfHeal:  "none",
+			expPrintStr:  "exclude disabled on pool due to [system] policy\nrebuild disabled on pool due to [system] policy\n",
+		},
+		"no flags": {
+			poolSelfHeal: "none",
+			sysSelfHeal:  "none",
+			expPrintStr:  "exclude disabled on pool due to [pool system] policies\nrebuild disabled on pool due to [pool system] policies\n",
```

Comment:
IMO it would be better to connect the strings with commas or something to indicate they're separate. "[pool system]" could be a little confusing. Maybe something like: "rebuild disabled on pool due to policies: system, pool"